Simcoe/Simcoe.swift

```diff
@@ -110,26 +110,6 @@ public final class Simcoe {
     public static func trackLocation(location: CLLocation, withAdditionalProperties properties: Properties?) {
         engine.trackLocation(location, withAdditionalProperties: properties)
     }
-    
-    /**
-     Logs the error with optional additional properties.
-     
-     - parameter error:      The error to log.
-     - parameter properties: The optional additional properties.
-     */
-    public static func logError(error: String, withAdditionalProperties properties: Properties? = nil) {
```

Comment:
i think we should leave this here, as it's generic enough and will likely be reused with other analytics providers